level2/load-balancer.tf

```diff
@@ -1,7 +1,73 @@
-module "lb" {
-  source = "../modules/lb"
+module "external-sg" {
+  source  = "terraform-aws-modules/security-group/aws"
+  version = "5.1.0"
 
-  vpc_id           = data.terraform_remote_state.level1.outputs.vpc_id
-  env_code         = var.env_code
-  public_subnet_id = data.terraform_remote_state.level1.outputs.public_subnet_id
+  name        = "${var.env_code}-external"
+  description = "Security group for allow http traffic to ELB and egress traffic to targets"
+  vpc_id      = data.terraform_remote_state.level1.outputs.vpc_id
+
+  ingress_with_cidr_blocks = [
```

Comment:
you didn't consider use of https and certificate manager to handle https request.
could you please add this suggestion to your PR.

Reply:
added now, can you please review. thank you